components/ui/Badge.tsx

```diff
@@ -1,10 +1,10 @@
 import { theme } from "@/theme"
-import { StyleSheet, View, ViewProps } from "react-native"
+import { StyleSheet, View, ViewProps, ViewStyle } from "react-native"
 import CustomText from "@/components/ui/CustomText"
-
+type BadgeVariants = "default" | "outline" | "primary"
```

Comment:
let's put this variant in the tyes folder, same as other variants. and also change the name to `BadgeVariant`